app/components/status/SystemServicesStatus.tsx

```diff
@@ -43,3 +46,21 @@ export default function SystemServicesStatus({
         />
     );
 }
+
+const mapStateToProps = (state: ReduxState) => {
+    return {
+        services: state.manager.clusterStatus.services,
+        isFetching: state.manager.clusterStatus.isFetching,
+        fetchingError: state.manager.clusterStatus.error
+    };
```

Comment:
WDYT about this version:
```suggestion
    const { services, isFetching, error: fetchingError } = state.manager.clusterStatus;
    return {
        services,
        isFetching,
        fetchingError
    };
```
?

Reply:
[451c5ac](https://github.com/cloudify-cosmo/cloudify-stage/pull/2310/commits/451c5ac140e05d704226f21113e4fd140c24b263)